filesystem/gpt_partition_table.ksy

```diff
@@ -0,0 +1,72 @@
+meta:
+  id: gpt_partition_table
+  endian: le
+  # spec from https://en.wikipedia.org/wiki/GUID_Partition_Table
+  # For now we only allow for 512 byte sectors.
+  # For 4096 byte sectors, change 0x200 to 0x1000 in instances.primary.pos, instances.backup.pos and types.partition_header.instances.entries.pos 
```

Comment:
We totally need a `license` to accept this. Also, `title` and `doc` at least at the top level are very welcome.

Reply:
Did some research how to write it, now incl. the correct license info.